.github/container/bump.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+## Parse command-line arguments
+
+usage() {
+cat <<EOF
+Usage: $0 [OPTION]...
+  -b, --only-bump-patches        If provided, update patch files and the patchlist in the manifest, but skip bumping refs
+  -i, --input-manifest  PATH     If set, will clean the patch dir. Default is not to clean
+  -h, --help                     Print usage.
+  -o, --output-manifest PATH     Use this if you don't want to update manifest in-place
```

Comment:
Update the description to something like: `Path to output manifest. Use this if you don't want to update manifest in-place`

Reply:
Done